src/index.html

```diff
@@ -7,8 +7,73 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+          rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home">
+        <img
+        src="images/logo.png"
+        alt="logo"
+        class="logopng">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item ">
+            <a href="#apple"
+            class="nav_link is-active">
+            APPLE
+            </a>
+          </li>
+          <li class="nav_item">
```

Comment:
Don't forget to add an empty line between multiline elements
```suggestion
          </li>
          
          <li class="nav_item">
```